vars/runPipeline.groovy

```diff
@@ -328,7 +347,13 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
 
     def dub_projects = [
         // sorted by test time slow to fast
-        "rejectedsoftware/vibe.d",
+        "vibe-d/vibe.d+libevent-base",
+        "vibe-d/vibe.d+libevent-examples",
+        "vibe-d/vibe.d+libevent-tests",
+        "vibe-d/vibe.d+vibe-core-base",
+        "vibe-d/vibe.d+vibe-core-examples",
+        "vibe-d/vibe.d+vibe-core-tests",
+        "vibe-d/vibe.d+libasync-base",
```

Comment:
Imho we should drop this too. At least until [event core](https://github.com/vibe-d/eventcore) supports it...